### ISSUE TITLE
[Tracker] Bolt 3.3 Release Blocking Issues

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#6001]
See also: https://github.com/bolt/bolt/milestone/21

## Open 

None at the moment.

### Feature/Bug PRs

None at the moment.

### Related

Bugs not necessarily related to 3.3, but ones that would be really nice to have fixed in the 3.3.0 release

None at the moment.

## Closed
 - #6691 — ~~[BUG] $app['query']->getContent when fetching content that has a relationship with self~~
 - #5885 — ~~Unable to delete records on the dashboard~~
 - #6278 — ~~Remove array_column; doesn't work for PHP 5.5/5.6~~
 - #6377 — ~~Sessions partially broken when serving from IP address~~
 - #6378 — ~~Repeater with required fields and empty first~~
 - #6413 — ~~Change of FieldType->getStorageType() crashes custom field types~~
 - #6420 — ~~Extensions break on missing `load()`~~
 - #6438 — ~~[3.3] Repeaters mismatch between slug and key~~
 - #6456 — ~~[3.3] Can't get template listing on themes with "links"~~
 - ~~Some form of resolution for #6472, #6473, #6474~~
 - #6473 — ~~Api compatibility with legacy content objects~~
 - #6474 — ~~Fix/clean up sub fields~~
 - #6437 — ~~[3.3] Regression: Twig functions defined extensions, added in content no longer work.~~ 
 - #6455 — ~~[3.3] Regression: No .htaccess throws exception when Apache checks are disabled.~~ 
 - #6504 — ~~Improving the wording and looks of the SecurityPolicy and Twig-in-content errors.~~ 
 - https://github.com/bolt/thumbs/pull/41  — ~~Thumbs: Check the id instead to make sure exif rotation works. (Note: the PR actually is OK, it's the tests that need fixing, because of the latest changes in Bolt/Filesystem)~~
 - #6382 — ~~Improving Controller Reusability (WIP solution: #6443 — TemplateView)~~
 - #6577 — ~~Tweaked Error/Exception Handler Configuration~~
 - #6580 — ~~[3.3] Nut IO BC & derps~~ 
 - https://github.com/bolt/docs/pull/674 — ~~Local extension migration guide. Needs some tweaking before we can merge in.~~ 
 - #6561 — ~~Custom sorting doesn't work for `type: select` (Fixed by #6638)~~
 - #6624 — ~~Fix TemplateView handling~~
 - #6645 — ~~Sessions starting before extensions registering~~
 - https://github.com/bolt/collection/pull/1 — ~~README & tagging a stable release~~
 - #6648 — ~~Smoother transition for "notfound_image" for bolt 3.2 -> 3.3~~ 
 - #6538 — ~~`ScriptHandler::installAssets` doesn't work if webdir is `.`.~~
 - #6582 — ~~[RFC] Remove `.bolt.yml` by default.~~
 - #6677 — ~~Corrupt images break "View Files" page.~~
 - #6762 — ~~Allow 'extensions_config' as a prefix for FilePermissions checks~~
 - #6767 — ~~Allow .bolt.php to return extension objects~~
 - #6797 — ~~Path customisation breaks on 3.3+ upgrades~~
  - #6782 — ~~[Bug] Custom Field Types Throw A Warning~~

## Moved

 - None

## Next 

- 3.4 Release tracker [#6412](https://github.com/bolt/bolt/issues/6412)

## Previous

- 3.2 Release tracker #5686

**Comment by bobdenotter** [comment]
Took the liberty of adding 6347 and 6348. :-)

**Comment by robsonsobral** [comment]
> Took the liberty of adding 6347 and 6348. :-)

Where's [6347[(https://github.com/bolt/bolt/issues/6347)?

**Comment by bobdenotter** [comment]
Sorry, can't type.. I meant 6437 and 6438 :-)

**Comment by GawainLynch** [comment]
> Sorry, can't type

You've been hanging around with me for too long … that's suallly [sic] my  problem.

**Comment by GawainLynch** [comment]
Released!
